lib/bolt/task.rb

```diff
@@ -58,7 +58,22 @@ def select_implementation(target, additional_features = [])
       impl['input_method'] = inmethod unless inmethod.nil?
 
       mfiles = impl.fetch('files', []) + metadata.fetch('files', [])
-      impl['files'] = mfiles.map { |file| { 'name' => file, 'path' => file_map[file] } } unless mfiles.empty?
+      unless mfiles.empty?
```

Comment:
does this check matter? It seems like we can rid of one level indentation otherwise.

Reply:
It affects other tests, and consumers will need to change slightly. But I initially did go without it.